From c5005870684d589868ab5e5710320e1286bb5572 Mon Sep 17 00:00:00 2001
From: Adrien Destugues <pulkomandy@gmail.com>
Date: Sun, 9 Aug 2015 22:31:25 +0200
Subject: Hack in Haiku support.

* Could not get LIBDIRARCH to work, so override it
* Add a prefix variable to the pkg-config file so it can be fixed by
* haikuporter

diff --git a/Makefile b/Makefile
index 52ae2a9..c787759 100644
--- a/Makefile
+++ b/Makefile
@@ -59,11 +59,11 @@ else
 BLDIR = $(abspath $(BUILDDIR))
 OBJDIR = $(BLDIR)/obj
 endif
-INCDIR = $(DESTDIR)$(PREFIX)/include
+INCDIR = $(DESTDIR)$(PREFIX)/develop/headers
 
 UNAME_S := $(shell uname -s)
 
-LIBDIRARCH ?= lib
+LIBDIRARCH ?= lib/x86
 # Uncomment the below line to installs x86_64 libs to lib64/ directory.
 # Or better, pass 'LIBDIRARCH=lib64' to 'make install/uninstall' via 'make.sh'.
 #LIBDIRARCH ?= lib64
@@ -457,6 +457,7 @@ define generate-pkgcfg
 	echo 'Name: capstone' > $(PKGCFGF)
 	echo 'Description: Capstone disassembly engine' >> $(PKGCFGF)
 	echo 'Version: $(PKG_VERSION)' >> $(PKGCFGF)
+	echo 'prefix=$(PREFIX)' >> $(PKGCFGF)
 	echo 'libdir=$(LIBDIR)' >> $(PKGCFGF)
 	echo 'includedir=$(INCDIR)/capstone' >> $(PKGCFGF)
 	echo 'archive=$${libdir}/libcapstone.a' >> $(PKGCFGF)
-- 
2.2.2

